TauAnalysis/MCEmbeddingTools/plugins/CaloCleaner.h

```diff
@@ -0,0 +1,145 @@
+/** \class CaloCleaner
+ *
+ * Clean collections of calorimeter recHits
+ * (detectors supported at the moment: EB/EE, HB/HE and HO)
+ * 
+ * \author Tomasz Maciej Frueboes;
+ *         Christian Veelken, LLR
+ *
+ * \version $Revision: 1.9 $
+ *
+ * $Id: CaloCleaner.h,v 1.9 2013/03/23 09:12:51 veelken Exp $
```

Comment:
text between "$" "$" is useful only for CVS. It has no meaning now in git and should be removed.